resources/metadata/abapEnvironmentCheckoutBranch.yaml

```diff
@@ -52,7 +52,16 @@ spec:
           - PARAMETERS
           - STAGES
           - STEPS
-        mandatory: true
+        mandatory: false
+      - name: repositoryNamesFiles
```

Comment:
abapEnvironmentCloneGitRepos uses the parameter "repositories". Type should be "string"
If you add the scope GENERAL for these steps, please add it also for CloneGitRepos
 ```
- name: repositories
        type: string
        description: Specifies a YAML file containing the repositories configuration
        scope:
          - PARAMETERS
          - STAGES
          - STEPS
        resourceRef:
          - name: commonPipelineEnvironment
            param: abap/repositories
```

Reply:
Parameter has been changed to "repositories" and was added to GENERAL section for CloneGitRepo step as well as the other steps